docs/userguide/modules/getting-started/partials/deploy-db-pgcluster.adoc

```diff
@@ -0,0 +1,51 @@
+== Create a PostgreSQL database instance
+
+The application is going to use to a PostgreSQL database backend which
+can be setup using the `v5` channel of
```

Comment:
do we really need to mention the channel? It is default one, so perhaps to avoid mentioning to avoid later changes, if their default channel changes.

Reply:
Some time ago we didn't mention the channel (it was before v5 was available) and let user to install the latest. Then we have the scenario working with the particular api. But then v5 has become available (with different api) and became the latest and it caused the scenario to stop working because the API changed. So, back then we decided to pin the version of the channel to keep it consistent with the api used by the quickstart. It was actually a request from the downstream docs to make it so.

Reply:
Hm, but that means that we should start catching regressions. IMHO the user guide should try to use the latest available versions. 

Do we have acceptance tests that the scenarios advertised in the quick start? If not, we should definitely add them within this PR.

Reply:
we do not have such scenarios yes... however, I plan to add it once the quickstart is aggreed upon and merged.